MediaGalleryUi/view/adminhtml/web/js/directory/directoryTree.js

```diff
@@ -203,21 +204,38 @@ define([
          *  Handle jstree events
          */
         initEvents: function () {
+            this.firejsTreeEvents();
             this.overrideMultiselectBehavior();
 
-            $(this.directoryTreeSelector).on('loaded.jstree', function () {
-                this.checkChipFiltersState();
-            }.bind(this));
-
             $(window).on('reload.MediaGallery', function () {
-                this.checkChipFiltersState();
```

Comment:
This removal has introduced back the problem when you open the media gallery from the `Select from Gallery` button and the catalog/category folder is selected, but when you go back and open the media gallery from the wysiwyg, the wysiwyg folder is not selected.